native/cocos/editor-support/spine/IkConstraintData.h

```diff
@@ -39,6 +39,7 @@ namespace spine {
 class BoneData;
 
 class SP_API IkConstraintData : public ConstraintData {
+    RTTI_DECL
```

Comment:
Why need it?

Reply:
Because we don't want the default c++ rtti support which will make the package size 20KB bigger.
Spine has already provoided RTTI strategy, so we use `RTTI_DECL`/`RTTI_IMPL` to mark the classes that need the RTTI information. RTTI information is used in getting a unique type id for a `instance` which like c++ `typeid(*pointer)`.

With the RTTI information, a c++ function return a `Base` pointer which points a `Sub` instance will be downcast to `Sub` JS object in emscripten layer. Otherwise, without the RTTI info, in JS layer, we only could see `Base` class.